resources/lang/en/datatable.php

```diff
@@ -64,4 +64,12 @@
     'multi_sort' => [
         'message' => 'Multiple sort is active',
     ],
+    'buttons-macros' => [
```

Comment:
Thanks for the PR! You can add these lines to other languages as well?

Reply:
Task has been completed [62f03df](https://github.com/Power-Components/livewire-powergrid/pull/1599/commits/62f03dfcc90742b1c514e9abb3851f445d1bd693)